resources/image.go

```diff
@@ -173,7 +171,7 @@ func (i *imageResource) cloneWithUpdates(u *transformationUpdate) (baseResource,
 // filter and returns the transformed image. If one of width or height is 0, the image aspect
 // ratio is preserved.
 func (i *imageResource) Resize(spec string) (resource.Image, error) {
-	conf, err := i.decodeImageConfig("resize", spec)
+	conf, err := i.decodeImageConfig("resize", spec, i.Format)
```

Comment:
`decodeImagConfig` is a method -- it has access to its `Format`, it should not be needed to send it as arg.

Reply:
Makes sense. I wanted to make it clear which fallback is being used, but it's a bit unclean. 
Fixed.